tests/run-pass/catch_panic.rs

```diff
@@ -0,0 +1,62 @@
+// error-pattern: thread 'main' panicked at 'Hello from panic: 0', tests/compile-fail/catch_panic.rs:35:5
```

Comment:
A run-pass test with an error-pattern...?

We have stdout files for tests that are expected to print something.

Reply:
(This is still open.)